src/types/AdsConsent.interface.ts

```diff
@@ -0,0 +1,365 @@
+   *
+   * @param geography The debug geography location.
+   */
+  setDebugGeography(
+    geography:
+      | AdsConsentDebugGeography['DISABLED']
+      | AdsConsentDebugGeography['EEA']
+      | AdsConsentDebugGeography['NOT_EEA'],
+  ): Promise<void>;
+
+  /**
+   * Manually update the consent status of the user.
+   *
+   * This method is used when providing your own means of user consent. If using the Google-rendered form via `showForm()`,
+   * the consent status is automatically set and calling this method is not required.
+   *
+   * This method can also be used to reset the consent status, by setting it to `AdsConsentStatus.UNKNOWN`, which may be useful in certain circumstances.
+   *
+   * #### Example
+   *
+   * ```js
+   * import { AdsConsent, AdsConsentStatus } from '@invertase/react-native-google-ads';
+   *
+   * // User accepted personalized ads
+   * await AdsConsent.setStatus(AdsConsentStatus.PERSONALIZED);
+   * ```
+   *
+   * @param status The user consent status.
+   */
+  setStatus(
+    status:
+      | AdsConsentStatus['UNKNOWN']
+      | AdsConsentStatus['NON_PERSONALIZED']
+      | AdsConsentStatus['PERSONALIZED'],
+  ): Promise<void>;
+
+  /**
+   * Returns the current consent status of the user.
+   *
+   * > The user consent status may change at any time, therefore don't reuse old values locally and always request the current value at any time consent is required.
+   *
+   * #### Example
+   *
+   * ```js
+   * import { AdsConsent } from '@invertase/react-native-google-ads';
+   *
+   * const status = await AdsConsent.getStatus();
+   * ```
+   */
+  getStatus(): Promise<
+    | AdsConsentStatus['UNKNOWN']
+    | AdsConsentStatus['NON_PERSONALIZED']
+    | AdsConsentStatus['PERSONALIZED']
+  >;
+
+  /**
+   * If a publisher is aware that the user is under the age of consent, all ad requests must set TFUA (Tag For Users
+   * Under the Age of consent in Europe). This setting takes effect for all future ad requests.
+   *
+   * Once the TFUA setting is enabled, the Google-rendered consent form will fail to load. All ad requests that include
+   * TFUA will be made ineligible for personalized advertising and remarketing. TFUA disables requests to third-party
+   * ad technology providers, such as ad measurement pixels and third-party ad servers.
+   *
+   * To remove TFUA from ad requests, set the value to `false`.
+   *
+   * #### Example
+   *
+   * ```js
+   * import { AdsConsent } from '@invertase/react-native-google-ads';
+   *
+   * // User is under age of consent
+   * await AdsConsent.setTagForUnderAgeOfConsent(true);
+   * ```
+   *
+   * @param tag The boolean value to tag for under age consent.
+   */
+  setTagForUnderAgeOfConsent(tag: boolean): Promise<void>;
+
+  /**
+   * If using a real device to test, ensure the device ID is provided to the Google Mobile Ads SDK so any mock debug locations
+   * can take effect.
+   *
+   * Emulators are automatically whitelisted and should require no action.
```

Comment:
```suggestion
   * Emulators are automatically on the allowlist and should require no action.
```

google's got a nifty inclusive language filter which makes it trivial to perform what is really one of the more basic gestures of inclusiveness, so I always act on it